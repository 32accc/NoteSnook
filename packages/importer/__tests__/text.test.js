const text = require("../providers/plaintext");

test("convert evernote note enex file", () => {
  let file = {
	  data:"Hello world",
	  fileName:"world.txt",
	  createdAt:0,
	  modifiedAt:0
  }

  expect(text.convert([file])).toMatchSnapshot("text-file");
});