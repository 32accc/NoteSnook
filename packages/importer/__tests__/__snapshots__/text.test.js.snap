// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert evernote note enex file: text-file 1`] = `
Array [
  Object {
    "content": Object {
      "data": "<p>Hello world</p>",
      "type": "text",
    },
    "dateCreated": 0,
    "dateEdited": 0,
    "title": "world",
  },
]
`;
